src/main/webapp/app/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/tutorial-groups-management/tutorial-groups-export-button/tutorial-groups-export-button.component.js

```diff
@@ -0,0 +1,67 @@
+import { __decorate, __metadata } from 'tslib';
```

Comment:
I think we don't want to use plain JavaScript. This file should be a typescript file